_sass/_settings--bootstrap.scss

```diff
@@ -4,9 +4,16 @@
  * to be used by Bootstrap to delivery its grid and
  * overall styles. A complete list of variables
  * can be found at: (http://getbootstrap.com/customize/#less-variables)
```

Comment:
Actually the link here is wrong, yours is better the comment that I set for a recent project was:

```
Add into this file the variables that are going
  to be used by Bootstrap to delivery its grid and
  overall styles. A complete list of variables
  can be found within the bootstrap module at node_modules
  within the _variables.scss file
```

I think we should use it and append the link you mentioned